### PR TITLE
Add configuration to expose content signing only

### PR DESCRIPTION
**Pull request by ncloudioj**
This implements feature for #317 

r? @oyiptong

### CODE REVIEW

File: splice/webapp.py [hunk 1]
```diff
@@ -5,9 +5,14 @@
 register_flask_restful = False
 
 
-def setup_routes(app):
+def setup_routes(app, content_signing_only):
```
Comment (oyiptong) [inline splice/webapp.py]: you don't need this, you can do `env.application.config`

File: splice/default_settings.py [hunk 2]
```diff
@@ -10,6 +10,7 @@ class DefaultConfig(object):
     """
     DEBUG = True
     APPLICATION_ROOT = None
+    CONTENT_SIGNING_ONLY = False
```
Comment (oyiptong) [inline splice/default_settings.py]: Might be better to have ALLOWED_APPS = ['tiles', 'signing']

File: splice/webapp.py [hunk 3]
```diff
@@ -5,9 +5,14 @@
...
     global register_flask_restful
 
+    import splice.web.api.content
```
Comment (oyiptong) [inline splice/webapp.py]: better to put a if statement for each type of API endpoint being registered.

e.g.
```python
if 'tiles' in app.config.ALLOWED_APPS:
    """do tiles API registration"""

if 'signing' in app.config.ALLOWED_APPS:
    import splice.web.api.content
    splice.web.api.content.register_routes(app)
```

Much better than short-circuiting the function

**Comment by oyiptong** [comment]
feedback+

File: splice/webapp.py (hunk 3 above)
Comment (ncloudioj) [inline splice/webapp.py]: Yes, that looks way better.

**Comment by ncloudioj** [comment]
Landed in cce7f46f02041ecda82b1fd531baf5618038fa9d

**Pull request closed without merging**